### PR TITLE
Remove the sync block on hubspot-template-3

### PR DESCRIPTION
**Pull request by chintans1**
We're looking to start using this template for the Domain Connect sync flow, therefore we're removing the `syncBlock` on it. For our sync requests, we will digitally sign the requests to still maintain the security aspect of the flow.

### CODE REVIEW

File: hubspot.com.hubspot-template-3.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
     "serviceId": "hubspot-template-3",
     "serviceName": "HubSpot CMS Site",
     "version": 1,
-    "syncBlock": true,
```
Comment (pawel-kow) [inline hubspot.com.hubspot-template-3.json:7]: The template in this form would not be secure if `syncBlock` is removed.
If you intend to use signing, please add  `syncPubKeyDomain` in the same PR.
You may also want to add `syncRedirectDomain` for the sync flow, if you plan to use redirects.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: hubspot.com.hubspot-template-3.json (hunk 1 above)
Comment (chintans1) [reply hubspot.com.hubspot-template-3.json:7]: Thanks Pawel! I just updated it so this should be good now. Let me know if you have any more concerns!

File: hubspot.com.hubspot-template-3.json [hunk 2]
```diff
@@ -3,8 +3,9 @@
     "providerName": "HubSpot",
     "serviceId": "hubspot-template-3",
     "serviceName": "HubSpot CMS Site",
+    "syncPubKeyDomain": "domainconnect.hubspot.com",
+    "syncRedirectDomain": "app.hubspot.com, app.hubspotqa.com, api.hubspot.com, api.hubspotqa.com",
     "version": 1,
```
Comment (pawel-kow) [inline hubspot.com.hubspot-template-3.json:8]: One more thing, please bump the version. Sorry I did not notice that before.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: hubspot.com.hubspot-template-3.json (hunk 2 above)
Comment (chintans1) [reply hubspot.com.hubspot-template-3.json]: Done :)

**Review by pawel-kow — APPROVED** [review]
Lgtm!

**Pull request merged**